async-request-reply/README.md

```diff
@@ -84,21 +111,58 @@ For more information about this pattern, see [Asynchronous Request-Reply pattern
 1. Delete all the Azure resources for this Cloud Async Pattern
 
    ```bash
-   az group delete -n rg-asyncrequestreply -y
+   az group delete -n ${RESOURCEGROUP} -y
    ```
 
 ### Running localy
```

Comment:
Should this come somewhere BEFORE the cleanup step, since it still uses Azure resources?  Maybe it's the last H3 before the cleanup H2?